s/frontend/models/commenting/commenting-model.test.ts

```diff
@@ -378,45 +378,6 @@ export default <Suite>{
 		},
 	},
 	"archiving comments": {
-		"a regular user": {
-
-			async "can archive their own comment, sees it disappear"() {
```

Comment:
let's not just delete the relevant tests -- let's adapt them instead

Reply:
Resolved!